ansible/roles/go-cli/tasks/deploy.yml

```diff
@@ -6,15 +6,39 @@
     path: "{{ go_cli_conf_dir }}"
     state: directory
 
+- name: "Ensure nginx directory for go-cli exists"
```

Comment:
all the ansible tasks names start with lower case so far - suggest changing case in this file.